src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector128_1.cs

```diff
@@ -692,6 +692,16 @@ private string ToString([StringSyntax(StringSyntaxAttribute.NumericFormat)] stri
         // New Surface Area
         //
 
+        static bool ISimdVector<Vector128<T>, T>.AnyWhereAllBitsSet(Vector128<T> vector)
+        {
+            return (Vector128.EqualsAny(vector, Vector128<T>.AllBitsSet));
```

Comment:
nit: unnecessary parens here and in `Any`

If you could fix that in a follow up PR, that'd be great (going to merge this)

Reply:
Sounds good! Will put it up later  today.